src/main.rs

```diff
@@ -242,6 +242,9 @@ async fn run_server(addr: SocketAddr) -> anyhow::Result<()> {
     // spawning a background task that will schedule the jobs
     // every JOB_SCHEDULING_CADENCE_IN_SECS
     task::spawn(async move {
+        if env::var_os("TRIAGEBOT_TEST_DISABLE_JOBS").is_some() {
+            return;
+        }
```

Comment:
I shuffled the spawning code to separate functions to hopefully make the structure a little clearer (the `run_server` function was starting to get a little long in my opinion).

I'd rather not mention `TRIAGEBOT_TEST_DISABLE_JOBS` in README.md, since users should never need to know about it specifically. I did add some documentation to `server_test/mod.rs` discussing scheduled jobs.